enzyme.config.js

```diff
@@ -0,0 +1,4 @@
+import { configure } from 'enzyme';
```

Comment:
I think there is no need for this file `enzyme.config.js`, the configurations therein are already implemented on `__test__/setup.js`

Reply:
Okay